### ISSUE TITLE
ReactiveUI.RxApp throws System.TypeInitializationException at start-up

### ISSUE THREAD

**Issue by davecader** [reactiveui/ReactiveUI#1670]
Recently upgraded our WPF application from ReactiveUI 7.4 to ReactiveUI 8.2.1. When the application starts there is a System.TypeInitializationException being generated from ReativeUI.RxApp. The inner exception reports that the System.Reactive.Windows.Threading.dll, Version 3.0.1000.0 could not be loaded. This file, and its dependencies appear to be deployed correctly.

A search revealed that there was some version discrepancy with the System.Reactive 3.1.1 NuGet package which we have attempted to resolve with binding redirects, but to no avail.  This has now become a blocking issue and we have no choice but to rollback to ReactiveUI 7.

**Comment by ThomasBoettcherHolschen** [comment]
I don't know if it helps you, but I've switched from Nuget To Paket https://fsprojects.github.io/Paket/

The Version of the Nuget-Package that will be included in your project can be pined.

The paket.dependency File would look like this:

```
framework: net471
redirects: on

source https://api.nuget.org/v3/index.json
nuget System.Reactive = 3.1.1
nuget ReactiveUI.WPF 
nuget ReactiveUI.Events.WPF
```

The paket.reference file like this :

```
ReactiveUI.WPF
ReactiveUI.Events.WPF
```

Calling Paket-Update confirms that Reactive.Ui Version 8.3.1 is used.

```
paket.exe update 
```

**Comment by glennawatson** [comment]
Pretty confident this is fixed in RxUI 8.4.4
